Lib/re.py

```diff
@@ -274,6 +274,7 @@ def escape(pattern):
 _MAXCACHE = 512
 def _compile(pattern, flags):
     # internal: compile pattern
+    flags = int(flags)
```

Comment:
The drawback of this change is that `flags=1.0` and `flags="0x1f"` are accepted now.

Reply:
Maybe use the .value attribute?

```
# convert RegexFlag enum to integer for performance
try: flags = flags.value
except AttributeError: pass
```

Would it be expensive to add a "isinstance(flags, RegexFlags)"?

```
if isinstance(flags, RegexFlags): flags = flags.value
```

Reply:
Check what is the fastest method.